example-ref-arch.yml

```diff
@@ -0,0 +1,477 @@
+  vip_admin_ip: x.x.x.x
+  vip_internal_ip: x.x.x.x
+  keepalived_internal_ipvs: 
+    - x.x.x.x
+  galera_ip: x.x.x.x
+  vip_admin_fqdn: osprv.somewhere.tld
+  vip_internal_fqdn: osprv.somewhere.tld
+  endpoint_protocol: http
+
+  ceilometer_admin_proto: http
+  ceilometer_internal_proto: http
+  ceilometer_public_proto: http
+  mgmt_names: ['ctrl01', 'ctrl02', 'ctrl03']
+  mgmt_internal_ips: ['x.x.x.x', 'x.x.x.x', 'x.x.x.x']
+  spof_cluster_members : 'ctrl01 ctrl02 ctrl03'
+  horizon_allowed_hosts:
+    - osprv.somewhere.tld
+
+  mongo_replset_members:
+    - ctrl01:27017
+    - ctrl02:27017
+    - ctrl03:27017
+
+  public_network: x.x.x.x/x
+  internal_network: x.x.x.x/x
+  admin_network: x.x.x.x/x
+  storage_network: x.x.x.x/x
+  tunnel_network: x.x.x.x/x
+
+  ntp_servers:
+    - 0.x.pool.ntp.org
+    - 1.x.pool.ntp.org
+    - 2.x.pool.ntp.org
+    - 3.x.pool.ntp.org
+
+  root_password: password
+  
+  public_netif: public
+  internal_netif: internal
+  admin_netif: admin
+  storage_netif: storage
+  lb_public_netif: lb_public
+  lb_internal_netif: lb_internal
+  external_netif: external
+  tunnel_netif: tunnel
+
+  public_netif_ip: "%{::ipaddress_public}"
+  internal_netif_ip: "%{::ipaddress_internal}"
+  admin_netif_ip: "%{::ipaddress_admin}"
+  storage_netif_ip: "%{::ipaddress_storage}"
+  tunnel_netif_ip: "%{::ipaddress_tunnel}"
+  external_netif_ip: "%{::ipaddress_external}"
+  lb_public_netif_ip: "%{::ipaddress_lb_public}"
+  lb_internal_netif_ip: "%{::ipaddress_lb_internal}"
+
+  keystone_db_user: user
+  keystone_db_password: password
+  ks_admin_email: keystone@somewhere.tld
+  ks_admin_password: password
+  ks_admin_tenant: admin
+  ks_admin_token: token
+
+  ks_swift_dispersion_password: dispersion_password
+  ks_swift_password: password
+  swift_hash_suffix: hash_suffic
+  swift_ring_container_device: sdX
+  swift_ring_account_device: sdX
+  swift_device_config_hash:
+    sdX:
+      mnt_base_dir: /srv/node
+   
+  mysql_root_password: password
+  mysql_sys_maint_user: sys-maint
+  mysql_sys_maint_password: password
+  galera_clustercheck_dbuser: clustercheckuser
+  galera_clustercheck_dbpassword: password
+  galera_master_name: ctrl01
+
+  db_allowed_hosts:
+    - ctrl%
+    - x.x.x.%
+
+  haproxy_auth: root:password
+
+  secret_key: key
+
+  rabbit_password: password
+  rabbit_host: x.x.x.x
+  rabbit_hosts:
+    - x.x.x.x:5672
+    - x.x.x.x:5672
+    - x.x.x.x:5672
+
+  ks_neutron_password: password
+  neutron_db_password: password
+  ks_neutron_admin_proto: https
+
+  neutron_tenant_network_types:
+    - vxlan
+  neutron_type_drivers:
+    - vxlan
+    - vlan
+  neutron_tunnel_types:
+    - vxlan
+  neutron_provider_bridge_mappings:
+    - network_name:br-pubX
+  neutron_provider_vlan_ranges:
+    - network_name:XXX:XXX
+  neutron_mechanism_drivers: openvswitch
+  neutron_plugin: ml2
+  neutron_tunnel_id_ranges: 1:10000
+  neutron_firewall_driver: neutron.agent.linux.iptables_firewall.OVSHybridIptablesFirew
```

Comment:
Err why? :(

Like having an example for ref-arch I thought it would be nice to have a vxlan example (because that's what people are using in real life).